acceptance/scenario_sample_observe_test.go

```diff
@@ -112,12 +112,12 @@ func TestAcc_Cmd_Scenario_Sample_Observe(t *testing.T) {
 							Scenario: &pb.Ref_Scenario{
 								Id: &pb.Scenario_ID{
 									Name:   "smoke",
-									Uid:    "21610357b0083126896a0072a429e677fd6381fa98e94b15ac2101f697100b42",
```

Comment:
This change is due to the fact that we now sort our matrix when we decode and filter it and before we did not do that.